great_expectations/data_context/store/ge_cloud_store_backend.py

```diff
@@ -89,7 +89,7 @@ def get_user_friendly_error_message(
     support_message = []
     response: requests.Response = http_exc.response
 
-    logger.warning(f"{http_exc.__class__.__name__}:{http_exc} - {response}")
+    logger.info(f"{http_exc.__class__.__name__}:{http_exc} - {response}")
```

Comment:
Feel bad about changing this but if set at `warning`, it results in double output during the migration process.

Reply:
instead of passing an fn, I decided to pass in a log level - my concern with using the fn is that we're explicitly referencing the logger defined in that file. The `logger` object will be different depending on where I call the method.

Reply:
Good call!